packages/tools/examples/videoTools/index.ts

```diff
@@ -1,4 +1,10 @@
-import { RenderingEngine, Types, Enums } from '@cornerstonejs/core';
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  eventTarget,
+  triggerEvent,
+} from '@cornerstonejs/core';
```

Comment:
in this example, next, previous does not work

Reply:
clear Range unpause the video which is not expected, not even sure what is clear range

There is also Add, Set, Start, End which either is confusing or don't work

Reply:
I prefer scroll to do scrolling through the times but right click is scroll which is unusuall

Reply:
Selection: 1.6 is confusing, You mean the annotation time or something?